src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
@keradus,
If people [installed php-cs-fixer globally](https://github.com/FriendsOfPHP/PHP-CS-Fixer#globally-manual), then that executable could be used in grumphp. This means that we cannot autoload the php-cs-fixer configuration class / interface at that point and we won't be able to check if the config contains a finder.

Maybe another path-mode could be added that only uses the files added to the cli command?